camel-example-spring-boot-actuator-http-metrics/src/main/java/sample/camel/MyRouteBuilder.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sample.camel;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.model.RouteDefinition;
+import org.apache.camel.model.rest.RestBindingMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MyRouteBuilder extends RouteBuilder {
+
+    @Override
+    public void configure() throws Exception {
+
+        // First, we have to configure our jetty component, which will be the rest
+        // in charge of querying the REST endpoints from actuator
+        restConfiguration().component("jetty")
+                .host("0.0.0.0")
+                .port(8080)
+                .bindingMode(RestBindingMode.json);
+
+        // First, let's show the routes we have exposed. Let's create a timer
+        // consumer that will only fire once and show us the exposed mappings
+        from("timer:queryTimer?repeatCount=1")
+                .to("rest:get:/actuator/mappings")
+                .unmarshal()
+                .json()
+                .to("log:INFO");
```

Comment:
You can use `multiline=true` option, which can be a bit easier to read.
```suggestion
                .to("log:INFO?multiline=true");
```

Reply:
Added it and prettyprinted it, but it doesn't improve "much"?

Like it doesn't pretty print past the first line. I think it's because it's not being mapped to any Java Class, I remember a similar issue with GSON in the past.

I added it to the metrics call as well. Makes it easier to read (much more because is a much less nested json object)

Commit is 96f127064d1e72ddd4dfb6b8f8c84851c14e1ba1